platforms/platform-alexa/src/output/templates/AudioPlayerPlayOutput.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+import { AudioItem, PlayBehavior, PlayBehaviorLike } from '../models';
+import _merge from 'lodash.merge';
+
+export interface AudioPlayerPlayOutputOptions extends OutputOptions {
+  playBehavior?: PlayBehaviorLike;
+  audioItem?: AudioItem;
+}
+
+@Output()
+export class AudioPlayerPlayOutput extends BaseOutput<AudioPlayerPlayOutputOptions> {
+  getDefaultOptions(): AudioPlayerPlayOutputOptions {
+    return {
+      playBehavior: PlayBehavior.ReplaceAll,
+      audioItem: {
+        stream: {
+          url: '',
+          token: '',
+          offsetInMilliseconds: 0,
+        },
+      },
+    };
+  }
+
+  build(): OutputTemplate | OutputTemplate[] {
+    if (this.options.audioItem) {
+      this.options.audioItem.stream.offsetInMilliseconds =
+        this.options.audioItem.stream.offsetInMilliseconds || 0;
```

Comment:
Is this step necessary? Because the default option `offsetInMilliseconds: 0` will be overridden anyways, if an offset is passed

Reply:
Yes
```return this.$send(AudioPlayerPlayOutput, {
      message: 'Starting audio',
      audioItem: {
        stream: {
          url: 'https://s3.amazonaws.com/jovo-songs/song1.mp3',
        },
      },
    });
    ``` 
    
    It isn't merged with the default value.

Reply:
Ok, why is it not being merged? Is this a bug?